static/js/src/interfaces/components/InterfaceDetails/InterfaceDetails.tsx

```diff
@@ -89,9 +90,11 @@ function InterfaceDetails({ interfaceItem }: Props) {
           {interfaceData?.name && <>{interfaceData?.name}</>}
           {!interfaceData?.name && interfaceName}
         </h1>
-        <p>
-          <Link to="/interfaces">See all interfaces</Link>
-        </p>
+          {showLink && (
```

Comment:
Is there a case when `showLink` wouldn't be `false`? `setShowLink` is never called.

Reply:
No, because we want to hide the link for the time being. And I agree that the way `useState `is used isn't ideal in this scenario. Then would it be okay if I remove the link itself?

Reply:
Yes, I think if the link isn't needed right now it could be removed, unless we know when we're going to use it.